graphql_compiler/schema_generation/schema_properties.py

```diff
@@ -111,12 +112,12 @@
 ORIENTDB_DATE_FORMAT = '%Y-%m-%d'
 
 
-def get_graphql_scalar_type_or_raise(property_name, property_type_id):
-    """Return the matching GraphQLScalarType for the property type id, asserting it exists."""
+def try_get_graphql_scalar_type(property_name, property_type_id):
+    """Return the matching GraphQLScalarType for the property type id or None."""
     if property_type_id not in ORIENTDB_TO_GRAPHQL_SCALARS:
-        raise AssertionError(u'Property "{}" has unsupported property type id: '
-                             u'{}'.format(property_name, property_type_id))
-    return ORIENTDB_TO_GRAPHQL_SCALARS[property_type_id]
+        warnings.warn(u'Ignoring property "{}" with unsupported property type id: 'u'{}'
+                      .format(property_name, property_type_id))
+    return ORIENTDB_TO_GRAPHQL_SCALARS.get(property_type_id, None)
```

Comment:
Let's not fetch data twice from the `ORIENTDB_TO_GRAPHQL_SCALARS` dict, that just opens us up to bugs. Instead, do `ORIENTDB_TO_GRAPHQL_SCALARS.get(property_type_id, None)` first, assign it to a variable, and check it for being equal to `None`, then warn.